src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.Cipher.cs

```diff
@@ -158,7 +158,7 @@ internal static unsafe bool EvpAeadCipherFinalEx(
                 notNullOutput = (stackalloc byte[1]).Slice(1);
             }
 
-            fixed (byte* pOutput = &MemoryMarshal.GetReference(notNullOutput))
+            fixed (byte* pOutput = notNullOutput)
```

Comment:
I think this change is introducing a bug. It looks like the code used `MemoryMarshal.GetReference` to avoid normalization of empty spans to null. You would have to also delete `.Slice(1)` above to make this change work.

Reply:
Addressed